### PR TITLE
Feature/desktop launch screen

### PR DESCRIPTION
**Pull request by chexee**
A few things still TODO:

- [ ] Detect OS and correct keyboard shortcut
- [ ] Add placeholder email body

### CODE REVIEW

**Comment by chexee** [comment]
screenshot 2016-07-20 18 23 32

File: app/web_modules/sourcegraph/desktop/DesktopHome.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+					See live examples, search code, and view inline
+					<br className={base["hidden-s"]} />&nbsp;documentation to write better code, faster
+				</Heading>
+
+				<img src={`${this.context.siteConfig.assetsRoot}/img/sg-desktop.gif`} width="356" title="Usage examples right in your editor" alt="Usage examples right in your editor" style={{maxWidth: "100%", display: "block", imageRendering: "pixelated"}} className={base.center}/>
+
+				<div className={base.mv4}>
+					<Heading level="5">Go definitions and usages as you code</Heading>
+					<p>
+						Install one of our editor integrations, and as you write Go code, this pane will update with contextually relevant information.
```
Comment (uforic) [inline app/web_modules/sourcegraph/desktop/DesktopHome.js]: We can be a stub link on integrations here, <a href="", and Renfred will ahve the integrations endpoint up by EOD.

File: app/web_modules/sourcegraph/desktop/DesktopHome.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+				<div className={base.mv4}>
+					<Heading level="5">Semantic, global code search</Heading>
+					<p>
+						Just hit {windowsShortcut} or click the search box at the top of this page to semantically search for functions and symbols.
+					</p>
+				</div>
+				<div className={base.mv4}>
+					<Heading level="5">Powerful search for your private code</Heading>
+					<p>
+						To enable semantic search and usage examples for your private code, authorize Sourcegraph to access your private repositories.
```
Comment (uforic) [inline app/web_modules/sourcegraph/desktop/DesktopHome.js]: We'll make this link to the /repositories view, maybe make this text this link? : authorize Sourcegraph to access your private repositories

Let's make it "index" your private repositories (per the conversation about how to not make it sound invasive)

Alternatively, "authorize Sourcegraph" could be the link text, to make it less of a chunk of text.

File: app/web_modules/sourcegraph/desktop/DesktopHome.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+					</Heading>
+					<p>
+						If the app is not working as expected, see our GitHub to:
+					</p>
+					<List className={base.mv3}>
+						<li><strong><a href="https://github.com/sourcegraph/sourcegraph-desktop#sourcegraph-desktop">Browse troubleshooting tips</a></strong></li>
+						<li><strong><a href="https://github.com/sourcegraph/sourcegraph-desktop/issues/new">File an issue</a></strong></li>
+					</List>
+					<p>
+						We love feedback! Shoot us an email at <strong><a href="mailto:support@sourcegraph.com?subject=Feedback for the Sourcegraph Desktop team&body=Editor of choice: %0D%0A%0D%0AOperating system:%0D%0A%0D%0AProgramming language:%0D%0A%0D%0AFeedback:%0D%0A%0D%0AP.S. Did you know we use Slack for feedback and bugs? Let us know if you'd like to join our Slack channel!">support@sourcegraph.com</a></strong> with ideas on how we can make Sourcegraph Desktop better.
```
Comment (uforic) [inline app/web_modules/sourcegraph/desktop/DesktopHome.js]: I screwed up on the copy here, a minor change:

Thanks for using Sourcegraph Desktop! Let use know what you think:

Feedback:

Editor of choice: 

Operating system:

Programming language:

P.S. Did you know we use Slack for feedback and bugs? Let us know if you'd like to join our channel!

**Comment by uforic** [comment]
This looks sick. A couple minor copy things, but otherwise let's merge this puppy in!

**Comment by uforic** [comment]
@nicot - Yo Nico, this is the endpoint where we are going to auth just to folks on the "beta" list, FYI!

File: app/web_modules/sourcegraph/desktop/DesktopHome.js (hunk 3 above)
Comment (chexee) [inline app/web_modules/sourcegraph/desktop/DesktopHome.js]: It's a little weird to include a "Thanks" message in a draft for an email they're writing to *us*. Would say the same thing for the PS, to be honest. We should just include these things in the copy of the paragraph.

**Comment by chexee** [comment]
@nicot @uforic Made some updates – go ahead and merge whenever you're ready :)

**Comment by uforic** [comment]
LGTM! tested locally. Merge when passes CircleCI, thanks @nicot and @chexee

**Pull request merged**